From: Alexander Gerasiov <gq@debian.org>
Subject: Fix builtin checking
 Ldapscripts uses "type" command to discover some builtins and throw security
 warnings if absent. But this checking works incorrectly in several locales:
 see #555561.
 This patch tries to fix this issue in most accurate way I found.
Forwarded: https://sourceforge.net/tracker/?func=detail&aid=2999925&group_id=156483&atid=799968

Index: ldapscripts/lib/runtime
===================================================================
--- ldapscripts.orig/lib/runtime	2009-12-03 18:34:43.000000000 +0300
+++ ldapscripts/lib/runtime	2009-12-03 18:37:54.000000000 +0300
@@ -65,7 +65,7 @@
 # Input : string to test ($1)
 # Output : true or false
 is_builtin () {
-  type "$1" 2>/dev/null | grep -qi 'built'
+  LANG=C type "$1" 2>/dev/null | grep -qi 'built'
 }
 
 # Logs a string to $LOGFILE
